src/composables/trade/useSor.ts

```diff
@@ -245,17 +245,28 @@ export default function useSor({
           tokenOutAddress.toLowerCase()
         );
 
-        const tokenInAmount = BigNumber.from(deltas[tokenInPosition]).abs();
-
-        const tokenOutAmount = BigNumber.from(deltas[tokenOutPosition]).abs();
+        let tokenInAmount = BigNumber.from(deltas[tokenInPosition]).abs();
+        let tokenOutAmount = BigNumber.from(deltas[tokenOutPosition]).abs();
 
         if (swapType === SwapType.SwapExactOut) {
```

Comment:
I don't think these two if statements are needed. Otherwise a exact in stETH trade amount won't be mutated.